cpp/core/config/GlutenConfig.cc

```diff
@@ -16,12 +16,28 @@
  */
 
 #include <jni.h>
-
+#include <optional>
+#include <regex>
 #include "compute/ProtobufUtils.h"
 #include "config.pb.h"
 #include "jni/JniError.h"
 
+namespace {
+
+const std::string REGEX_REDACT_KEY = "spark.gluten.redaction.regex";
```

Comment:
move config key definition to GlutenConfig.h

Reply:
and follow others naming.

Reply:
This will be changed, we will no longer rely on this config

Reply:
All configs from GlutenConfig.scala should mapping to GlutenConfig.h/VeloxConfig.h, let's keep config consistent.